ansible/roles/tenderdash/tasks/main.yml

```diff
@@ -50,3 +50,32 @@
     state: present
     pull: always
     recreate: always
+
+- name: Configure logrotate for Tenderdash logs
+  ansible.builtin.copy:
+    dest: /etc/logrotate.d/tenderdash
+    content: |
+      {{ tenderdash_compose_path }}/tenderdash/logs/*.log {
+          daily
+          rotate 3
+          compress
+          delaycompress
+          missingok
+          notifempty
+          create 0644 {{ tenderdash_uid }} {{ tenderdash_gid }}
```

Comment:
why `0644` and not `0640`? World-readable logs are a potential security risk  (but sometimes it's needed, eg. when Docker needs to use it - please double-check).

Reply:
fixed